### PR TITLE
Mve2 home page

### PR DESCRIPTION
**Pull request by ethanolmethanol**

### CODE REVIEW

File: Pipfile
Comment (romann-broque) [inline Pipfile:1]: Goodbye!

File: Pipfile.lock
Comment (romann-broque) [inline Pipfile.lock:1]: Goodbye!

File: back/transcendence_django/back_user/urls.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from back_auth.views import is_logged_view
+from django.urls import path
+
+from .views import username_view
+
+urlpatterns = [
+    path("is_logged/", is_logged_view, name="is_logged"),
```
Comment (romann-broque) [inline back/transcendence_django/back_user/urls.py:7]: remove the is_logged part in the back_user

File: front/src/app/pages/home-page/banner/banner.component.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  right: 0;
+  background-color: black;
+  color: white;
+  padding: 10px;
+  z-index: 1000;
+}
+
+.banner-title {
+  font-size: 3em;
+  /* margin-bottom: 60px; */
```
Comment (romann-broque) [inline front/src/app/pages/home-page/banner/banner.component.css:17]: remove the comment

File: front/src/app/pages/home-page/banner/banner.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div color="primary" class="banner">
+  <button class="profile-button" routerLink="/profile">Profile</button>
+  <h1 class="banner-title">transcendence</h1>
```
Comment (romann-broque) [inline front/src/app/pages/home-page/banner/banner.component.html:3]: transcendence -> Transcendence

File: front/src/app/pages/home-page/home-page.component.html [hunk 4]
```diff
@@ -1,2 +1,8 @@
-<p>home-page works!</p>
-<app-logout></app-logout>
+<body>
+  <div class="home-page-container">
+    <app-banner></app-banner>
+    <div class="welcome"><h3>Welcome, {{username}}!</h3></div>
```
Comment (romann-broque) [inline front/src/app/pages/home-page/home-page.component.html:4]: add an ngif condition to avoid empty username display

**Review by opaleboreale — APPROVED** [review]

**Review by romann-broque — CHANGES_REQUESTED** [review]

File: front/src/app/pages/home-page/home-page.component.html (hunk 4 above)
Comment (ethanolmethanol) [reply front/src/app/pages/home-page/home-page.component.html:4]: No <3

**Review by romann-broque — APPROVED** [review]
Okay

**Pull request merged**